### DISCUSSION TITLE
Missing pictures and "select all" button

### DISCUSSION THREAD

**Discussion by Bartho1omeus** [zencq/NomNom#80] [Q&A] [answered]
Hi there cengelha, 

First of all thank you for creating this tool that enables us to modify our nms console save game. 
I've been using it for a bit now and the things I'm missing are the pictures for the items that the other nms save editor for pc does have, also a "select all" button for the catalogue items and the words would be useful. Do you have any idea when you are going to release a new version?

**Comment by cengelha** [comment, marked as answer]
Hi @Bartho1omeus,

thanks for using NomNom! :)

I'm currently working on a bigger update that will improve a lot of things (have a look at the roadmap). The things you mentioned will be part of it. 
But don't expect any new features or bigger changes until that one is released.

**Reply by Bartho1omeus** [reply]
Thank you for your response, the editor as it currently is is working fine with the features now implemented in it. I haven't encountered any major bugs except for a few freezes of the program.
